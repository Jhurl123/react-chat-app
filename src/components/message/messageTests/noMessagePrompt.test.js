import React from 'react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import NoMessagePrompt from '../noMessagePrompt'

describe('No Message Prompt tests', () => {

  it('renders the ui as expected', () => {
    const tree = renderer
    .create(<NoMessagePrompt />)
    .toJSON();
    expect(tree).toMatchSnapshot();  
  });

  it('renders the UI without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NoMessagePrompt />, div);
    ReactDOM.unmountComponentAtNode(div);
  })

})