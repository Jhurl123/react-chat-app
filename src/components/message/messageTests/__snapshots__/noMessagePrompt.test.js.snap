// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Message Prompt tests renders the ui as expected 1`] = `
<div>
  <h2
    className="makeStyles-noPrompt-1"
  />
</div>
`;
